org.eclipse.winery.topologymodeler.ui/src/app/palette/palette.component.ts

```diff
@@ -136,12 +136,12 @@ export class PaletteComponent implements OnInit, OnDestroy {
      * @param $event
      */
     generateNewNode($event): void {
-        const left = ($event.pageX - this.newNodePositionOffsetX).toString();
-        const top = ($event.pageY - this.newNodePositionOffsetY).toString();
+        const x = ($event.pageX - this.newNodePositionOffsetX).toString();
```

Comment:
Why do you convert the number to string? (same question for y)

Reply:
Resulted in early development, corrected it to number